Branch-TestBed/Branch-TestBed/AppDelegate.m

```diff
@@ -42,7 +42,7 @@ - (BOOL)application:(UIApplication *)application
     //[branch validateSDKIntegration];
 
     // Check for Apple Search Ad attribution (trade-off: slows down app startup):
-    [branch delayInitToCheckForSearchAds];
+    //[branch delayInitToCheckForSearchAds];
```

Comment:
Intended?

Reply:
Yes.  By default apple always sends a test payload and I was getting tired of seeing it.